assets/src/edit-story/components/keyboardShortcutsMenu/shortcutMenu.js

```diff
@@ -17,153 +17,147 @@
 /**
  * External dependencies
  */
-import { Fragment, useMemo, useCallback, useEffect, useRef } from 'react';
+import styled from 'styled-components';
+import {
+  useCallback,
+  useEffect,
+  useRef,
+  useLayoutEffect,
+  useState,
+} from 'react';
 import PropTypes from 'prop-types';
 import { __ } from '@web-stories-wp/i18n';
+import { v4 as uuidv4 } from 'uuid';
 
 /**
  * Internal dependencies
  */
-import { Close as CloseIcon } from '../../icons';
-import { useKeyDownEffect } from '../../../design-system';
-import getKeyboardShortcuts from './getKeyboardShortcuts';
-import ShortcutLabel from './shortcutLabel';
 import {
-  Container,
-  PanelsWrapper,
-  Panel,
-  HeaderWrapper,
-  HeaderRow,
-  MenuHeaderContainer,
-  MenuHeader,
-  SectionHeader,
-  SectionWrapper,
-  SectionContent,
-  ContentWrapper,
-  Column,
-  Label,
-  PanelLabel,
-  CloseButton,
-} from './styled';
-import { TOGGLE_SHORTCUTS_MENU } from './constants';
+  Button,
+  BUTTON_SIZES,
+  BUTTON_TYPES,
+  BUTTON_VARIANTS,
+  Icons,
+  useKeyDownEffect,
+} from '../../../design-system';
+import { isKeyboardUser } from '../../utils/keyboardOnlyOutline';
+import useFocusTrapping from '../../utils/useFocusTrapping';
+import { ADMIN_TOOLBAR_HEIGHT, HEADER_HEIGHT } from '../../constants';
+import HeaderShortcut from './headerShortcut';
+import LandmarkShortcuts from './landmarkShortcuts';
+import RegularShortcuts from './regularShortcuts';
+import { TOGGLE_SHORTCUTS_MENU, TOP_MARGIN } from './constants';
 
-function ShortcutMenu({ toggleMenu }) {
-  const containerRef = useRef();
-  const closeRef = useRef();
-  const keyboardShortcuts = useMemo(() => getKeyboardShortcuts(), []);
+const BORDER_WIDTH = 1;
+const Container = styled.div`
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  width: 352px;
+  border-radius: ${({ theme }) => theme.borders.radius.small};
+  background-color: ${({ theme }) => theme.colors.bg.primary};
+  border: ${BORDER_WIDTH}px solid
+    ${({ theme }) => theme.colors.border.defaultNormal};
```

Comment:
Here, too, the border color seems incorrect (should be something with `white16`).

Reply:
As [mentioned](https://github.com/google/web-stories-wp/pull/6403#discussion_r576920962), yes.